.circleci/config.yml

```diff
@@ -72,6 +72,13 @@ workflows:
           os: windows
           requires:
             - release-management/test-package
+      - release-management/run-command-keyword:
```

Comment:
it might be good to give this a descriptive `name` (ex: `sandbox nuts`) so it doesn't show up in circle as `run-command-keyword` 

especially if you had several of them OR someone's looking to see what failed.